libs/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart.component.ts

```diff
@@ -40,12 +40,20 @@ export class FhirChartComponent implements OnInit {
 
   @Input() emptyMessage: string = 'No data';
 
-  constructor(private readonly configService: FhirChartConfigurationService, public layerManager: DataLayerManagerService) {}
+  private readonly keyboardListener: (event: KeyboardEvent) => void;
+
+  constructor(
+    public configService: FhirChartConfigurationService,
+    public layerManager: DataLayerManagerService,
+  ) {
+    this.keyboardListener = this.handleKeyboardZoomAndPan.bind(this);
```

Comment:
It's a better practice to define callbacks as arrow function properties instead of binding a method. You can simplify it to:
```
private readonly keboardListener = (event: KeyboardEvent) => {
  // body of handleKeyboardZoomAndPan method
}
```